.github/workflows/pr-preview.yml

```
@@ -0,0 +1,314 @@
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Create EAS Build
+        run: |
+          # Create build with PR-specific message
+          eas build --platform ios --profile preview --non-interactive --message "PR #${{ github.event.number }}: ${{ github.event.pull_request.title }}"
+
+      - name: Comment on PR - Build Started
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)